src/plugins/data/public/search/aggs/buckets/terms.ts

```diff
@@ -60,6 +60,21 @@ export interface TermsBucketAggDependencies {
   getInternalStartServices: GetInternalStartServicesFn;
 }
 
+export interface AggParamsTerms {
+  field: string;
+  order: 'asc' | 'desc';
+  orderBy: string;
+  orderAgg?: AggConfigJson;
+  size?: number;
+  missingBucket?: boolean;
+  missingBucketLabel?: string;
+  otherBucket?: boolean;
+  otherBucketLabel?: string;
+  // advanced
+  exclude?: string;
```

Comment:
these are common for all agg types right ?

Reply:
no, exclude and include are only specifically surfaced in the ui for some agg types (terms & significant terms i think).